kiwi/partitioner/gpt.py

```diff
@@ -79,3 +79,8 @@ def set_flag(self, partition_id, flag_name):
             )
         else:
             log.warning('Flag %s ignored on GPT', flag_name)
+
+    def set_hybrid_mbr(self):
+        Command.run(
+            ['sgdisk', '-h', '1:2:3', self.disk_device]
```

Comment:
And what happens on disk resize at first boot? Does the hybrid GPT get preserved?